src/Sulu/Bundle/AdminBundle/Admin/AdminPool.php

```diff
@@ -52,7 +55,7 @@ public function getNavigation()
         /** @var Navigation $navigation */
         $navigation = null;
         foreach ($this->pool as $admin) {
-            /** @var Admin $admin */
```

Comment:
that is not good (= intellij needs this double star to know that is phpdoc-information